keyboards/masterworks/classy_tkl/rev_a/rev_a.h

```diff
@@ -0,0 +1,79 @@
+    k30,      k31, k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B,           k3C,                     \
+    k40, k41, k42, k43, k44, k45, k46, k47, k48, k49, k4A, k4B,           k4C, k4D,           k4F,      \
+    k50, k52, k53,                     k57,                          k5B, k5C, k5D,      k5E, k5F, k5G  \
+) \
+{ \
+    { k00, KC_NO, k02, k03, k04,   k05,   k06,   k07, k08,   k09,   k0A,   k0B, k0C,   k0D, k0E,   k0F,   k0G },   \
+    { k10, k11,   k12, k13, k14,   k15,   k16,   k17, k18,   k19,   k1A,   k1B, k1C,   k1D, k1E,   k1F,   k1G },   \
+    { k20, k21,   k22, k23, k24,   k25,   k26,   k27, k28,   k29,   k2A,   k2B, k2C,   k2D, k2E,   k2F,   k2G },   \
+    { k30, k31,   k32, k33, k34,   k35,   k36,   k37, k38,   k39,   k3A,   k3B, k3C,   k3D, KC_NO, KC_NO, KC_NO }, \
+    { k40, k41,   k42, k43, k44,   k45,   k46,   k47, k48,   k49,   k4A,   k4B, k4C,   k4D, KC_NO, k4F,   KC_NO }, \
+    { k50, KC_NO, k52, k53, KC_NO, KC_NO, KC_NO, k57, KC_NO, KC_NO, KC_NO, k5B, k5C,   k5D, k5E,   k5F,   k5G },   \
+}
+
+#define LAYOUT_tkl_ansi_wkl( \
+    k00,      k02, k03, k04, k05,      k06, k07, k08, k09,      k0A, k0B, k0C, k0D,      k0E, k0F, k0G, \
+    k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, k1C,           k3D,      k1E, k1F, k1G, \
+    k20,      k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, k2C,      k2D,      k2E, k2F, k2G, \
+    k30,      k31, k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B,           k3C,                     \
+    k40,      k42, k43, k44, k45, k46, k47, k48, k49, k4A, k4B,                k4C,           k4F,      \
+    k50,      k53,                     k57,                          k5B,      k5D,      k5E, k5F, k5G  \
+) \
+{ \
+    { k00, KC_NO, k02,   k03, k04,   k05,   k06,   k07, k08,   k09,   k0A,   k0B, k0C,   k0D,   k0E,   k0F,   k0G },   \
+    { k10, k11,   k12,   k13, k14,   k15,   k16,   k17, k18,   k19,   k1A,   k1B, k1C,   KC_NO, k1E,   k1F,   k1G },   \
+    { k20, k21,   k22,   k23, k24,   k25,   k26,   k27, k28,   k29,   k2A,   k2B, k2C,   k2D,   k2E,   k2F,   k2G },   \
+    { k30, k31,   k32,   k33, k34,   k35,   k36,   k37, k38,   k39,   k3A,   k3B, k3C,   k3D,   KC_NO, KC_NO, KC_NO }, \
+    { k40, KC_NO, k42,   k43, k44,   k45,   k46,   k47, k48,   k49,   k4A,   k4B, k4C,   KC_NO, KC_NO, k4F,   KC_NO }, \
+    { k50, KC_NO, KC_NO, k53, KC_NO, KC_NO, KC_NO, k57, KC_NO, KC_NO, KC_NO, k5B, KC_NO, k5D,   k5E,   k5F,   k5G },   \
+}
+
+#define LAYOUT_tkl_iso_wkl( \
+    k00,      k02, k03, k04, k05,      k06, k07, k08, k09,      k0A, k0B, k0C, k0D,      k0E, k0F, k0G, \
+    k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, k1C,           k3D,      k1E, k1F, k1G, \
+    k20,      k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, k2C,      k2D,      k2E, k2F, k2G, \
+    k30,      k31, k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C,                               \
```

Comment:
Based on the ISO keymap you've provided, I think this needs to be:

```suggestion
    k20,      k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, k2C,                k2E, k2F, k2G, \
    k30,      k31, k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C,      k2D,                     \
```

QMK custom is that ISO Enter is to the end of the home row.